unity/Assets/Maroon/reusableScripts/Config/ConfigLoader.cs

```diff
@@ -0,0 +1,132 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Networking;
+using Newtonsoft.Json;
+using UnityEngine.Events;
+using SimpleJSON;
+using System.IO;
+using System.Linq;
```

Comment:
I don't think any of those are unnecessary. They are just shown as unnecessary, since their functions are used in #if - blocks. I don't know if it's good practise, but I will also put the usings in #if blocks now :)

Reply:
ah sorry about that then, thought they would be redundant, but thanks, sounds good 👍